app/controllers/content_pages_controller.rb

```diff
@@ -2,7 +2,7 @@ class ContentPagesController < ApplicationController
   before_action :authenticate_registered_user!
 
```

Comment:
This doesn't look right - why was `training_module` removed?

Reply:
https://rubystyle.guide/#hash-literal-values

Reply:
I agree, it is something new to get used to.

Reply:
Yikees!

Reply:
I think there is a difference between a hash and keyword arguments.  How does the method know what value you to pass in?  Some ruby magic under the covers?

Reply:
Basically, if your assignment is `x(foo: foo)` you can now do `x(foo:) and it will assume the `foo:` is referring to a variable of the same name. Not sure I like this one. It feels a bit to me like using `%i[a b]` when the more explicit `[:a, :b]` seems clearer to me. Shortcut for shortcut's sake. ... like HAML and Slim :smile: